opta/one_time.py

```diff
@@ -0,0 +1,28 @@
+import os
+import sys
+from pathlib import Path
+
+from opta.constants import CI, ONE_TIME_LAUNCH, one_time_run
+
+
+def one_time() -> None:
+    if os.environ.get(CI) is not None or os.environ.get(ONE_TIME_LAUNCH):
+        return
+
+    if Path(one_time_run).is_file():
+        return
+
+    try:
+        open(one_time_run, "w").close()
+    except:  # noqa: E722
+        sys.exit(1)
+
+    print(
+        "\nHi there, thanks for Using Opta.\nWe hope you have a great experience using Opta."
+    )
+
+    slack_prompt()
+
+
+def slack_prompt() -> None:
```

Comment:
why make a function that is only used once with a single line?

Reply:
If some other functionalities are required to be done, they can all be done in the method `one_time`. 
In order to not have a very cluttered method, having single line one time functions made sense.

Reply:
Removed as per request.